static/openapi.yaml

```diff
@@ -1430,14 +1430,20 @@ components:
           description: A list of the categories that the project is in
         client_side:
           type: string
-          enum: [required, optional, unsupported]
+          enum: [required, optional, unsupported, unknown]
           description: The client side support of the project
           example: required
         server_side:
           type: string
-          enum: [required, optional, unsupported]
+          enum: [required, optional, unsupported, unknown]
           description: The server side support of the project
           example: optional
+      required:
+        - title
+        - description
+        - categories
+        - client_side
+        - server_side
```

Comment:
I intentionally moved these out of BaseProject because it looked like the PATCH request required most everything when making any patch. None of the properties should be required for patch requests.

Reply:
Mh, required has to be defined where the property is defined, therefore moving them out from `BaseProject` is the same es removing them.

Reply:
Has to be? It works fine on the site - https://docs.modrinth.com/api-spec/#operation/getProject
I guess what the site accepts and what the official spec considers valid is different

Reply:
Yeah, the way you did it is probably the only solution here. I'll revert the changes affecting the patch request and double check if everything is correct - it is quite hard to keep track with this split.